native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -6,6 +6,7 @@
 #include <limits>
 #include <map>
 #include <utility>
+#include <set>
```

Comment:
do you really need `std::set` ? everywhere it is mentionned a list; `std::vector` is not enough?

Reply:
I'm using `std::set` because my understanding is that lookup (during decoding) will be faster than any other data structure. If this isn't correct, I'm glad to change to something else.

Reply:
That sounds good.